build.gradle

```diff
@@ -1,5 +1,5 @@
 plugins {
-    id 'java'
+    id 'java-library'
```

Comment:
Why `java-library`? It does not publish any artifacts

Reply:
Why not?